pkg/resourcemonitor/podresourcesscanner_test.go

```diff
@@ -499,14 +525,20 @@ func TestPodScanner(t *testing.T) {
 			}
 			mockAPIHelper.On("GetClient").Return(mockClient, nil)
 			mockAPIHelper.On("GetPod", mock.AnythingOfType("*kubernetes.Clientset"), "default", "test-pod-0").Return(pod, nil)
-			res, err := resScan.Scan()
+			res, annotations, err := resScan.Scan()
 
 			Convey("Error is nil", func() {
 				So(err, ShouldBeNil)
 			})
 			Convey("Return PodResources should have values", func() {
 				So(len(res), ShouldBeGreaterThan, 0)
 			})
+			Convey("Return Annotations should have pod fingerprint annotation with proper value", func() {
+				So(len(annotations), ShouldEqual, 1)
+
+				expectedFingerprint := "pfp0v001fe53c4dbd2c5f4a0" // precomputed manually
```

Comment:
hard-earned lesson from RTE: we should document which pods (namespace/name) we expect which lead to this fingerprint. Or, if possible, compute it in the tests.

Reply:
(here and also in all the instances below)

Reply:
I agree on the documentation part but how could we compute the result if we are testing the compute algorithm itself?

Reply:
the compute algorithm itself is encapsulated in the `podfingerprint` package, isn't it?
In order to make the tests debuggable is (much much) better to document/show the expectations in terms of pod running on nodes. How this set translate in which fingerprint value is the `podfingerprint` package job indeed.

Reply:
I have changed to compute the expected fingerprint using the list of expected pods. 
Also I have documented that fingerprint only depends on namespace and name of the running pods in a node.